frontend/src/components/chat/ChannelView.css

```
@@ -64,6 +64,68 @@
+.ch-archive-channel__button--confirm {
+  border-color: var(--status-error);
+  color: var(--status-error);
+}
+
+.ch-archive-channel__button:disabled {
+  opacity: 0.55;
+  cursor: not-allowed;
+}
+
+.ch-archive-channel__progress {
+  color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)